releases/release-1.12/release-1.12.md

```diff
@@ -0,0 +1,135 @@
+# Kubernetes 1.12 Release Schedule
+
+*Handy Links*
+
+* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.12/release_team.md)
+* Zoom: join Google Group to receive meeting invite
+* [Slack](https://kubernetes.slack.com/messages/sig-release/)
+* [Forum](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
+* [Feature Tracking Sheet](http://bit.ly/k8s112-features) **TODO: needs created, and short url'd**
```

Comment:
consider removing this since the tracker is alive now ?

Reply:
+1, nice catch!

Reply:
fixed